app/assets/stylesheets/layout.scss

```diff
@@ -2770,7 +2770,11 @@ table {
     }
 
     a:hover {
-      box-shadow: 0 0 12px 0 rgba(0, 0, 0, 0.2);
+
```

Comment:
Hi @PierreMesure! Works perfectly! 👌

Only by convention, can you please change the order?

```
a:hover {
  text-decoration: none;

  &:not(.see-all) {
    box-shadow: 0 0 12px 0 rgba(0, 0, 0, 0.2);
  }
```

So `text-decoration: none;` doesn't get left alone 😜 

Many thanks! 🙏